mycroft/skills/common_iot_skill.py

```diff
@@ -305,6 +355,16 @@ def register_entities_and_scenes(self):
         self._register_words(self.get_entities(), ENTITY)
         self._register_words(self.get_scenes(), SCENE)
 
+    @property
+    def supported_request_version(self) -> IoTRequestVersion:
+        """
+        Get the supported IoTRequestVersion
+
+        By default, this returns IoTRequestVersion.V1. Subclasses
+        should override this to indicate higher levels of support.
+        """
```

Comment:
Should the supported features for each version be described here (or perhaps up in the version enum) so it's included in the readthedocs documentation?